### ISSUE TITLE
Error starting cluster

### ISSUE THREAD

**Issue by D-marina** [ipython/ipyparallel#228]
Hello,

I tried to use several engines in jupyter and I followed this steps:

($ is my prompt)
1.- $jupyter serverextension enable --py ipyparallel
2.- $jupyter nbextension install --py ipyparallel
3.- $jupyter nbextension enable --py ipyparallel
4.- $jupyter nbextension enable ipyparallel --py

then I opened a browser, I clicked in IPython Clusters's tab, selected 2 engines and I pressed start. But in status I got "error starting cluster". I don't know if this issue is solved but I can't solve this problem.

I post this message in "jupyter-drive" and the told me to post here. Thank you very much.

In the console, I get the following:
```

Enabling: ipyparallel.nbextension
- Writing config: /home/daniel/.jupyter
    - Validating...
      ipyparallel.nbextension  OK
Installing /home/daniel/anaconda3/lib/python3.6/site-packages/ipyparallel/nbextension/static -> ipyparallel
Up to date: /usr/local/share/jupyter/nbextensions/ipyparallel/clusterlist.js
Up to date: /usr/local/share/jupyter/nbextensions/ipyparallel/clusterlist.css
Up to date: /usr/local/share/jupyter/nbextensions/ipyparallel/main.js
- Validating: OK

    To initialize this nbextension in the browser every time the notebook (or other app) loads:
    
          jupyter nbextension enable ipyparallel --py
    
Enabling tree extension ipyparallel/main...
      - Validating: OK
Enabling tree extension ipyparallel/main...
      - Validating: OK
[I 18:08:35.771 NotebookApp] The port 8888 is already in use, trying another port.
[I 18:08:35.772 NotebookApp] The port 8889 is already in use, trying another port.
[I 18:08:35.772 NotebookApp] The port 8890 is already in use, trying another port.
[I 18:08:35.796 NotebookApp] Loading IPython parallel extension
[I 18:08:35.803 NotebookApp] Serving notebooks from local directory: /home/daniel
[I 18:08:35.803 NotebookApp] 0 active kernels 
[I 18:08:35.803 NotebookApp] The Jupyter Notebook is running at: http://localhost:8891/?token=3b128095b6543d775871138eac300a7a6aee6fe4c09c077c
[I 18:08:35.803 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 18:08:35.804 NotebookApp] 
    
    Copy/paste this URL into your browser when you connect for the first time,
    to login with a token:
        http://localhost:8891/?token=3b128095b6543d775871138eac300a7a6aee6fe4c09c077c
[I 18:08:58.059 NotebookApp] 302 GET /?token=3b128095b6543d775871138eac300a7a6aee6fe4c09c077c (127.0.0.1) 2.51ms
[W 18:09:03.348 NotebookApp] 403 POST /clusters/default/start (127.0.0.1): '_xsrf' argument missing from POST
[W 18:09:03.357 NotebookApp] 403 POST /clusters/default/start (127.0.0.1) 9.40ms 
```referer=http://localhost:8891/tree?token=3b128095b6543d775871138eac300a7a6aee6fe4c09c077c

**Comment by minrk** [comment]
@D-marina what version of ipyparallel? This should have been fixed in 6.0.

**Comment by jeiros** [comment]
Hi,
I'm seeing the same error. I'm on ipyparallel version 5.2.0 but I can't upgrade it:
```bash
$ conda update ipyparallel
Fetching package metadata ...........
Solving package specifications: .

# All requested packages already installed.
# packages in environment at /Users/Juan/anaconda/envs/numba_tutorial:
#
ipyparallel               5.2.0                    py35_0
$ conda search ipyparallel
Fetching package metadata ...........
ipyparallel                  4.0.0                    py27_0  defaults
                             4.0.0                    py34_0  defaults
                             4.0.1                    py34_0  defaults
                             4.0.1                    py27_0  defaults
                             4.0.2                    py34_0  defaults
                             4.0.2                    py27_0  defaults
                             4.0.2                    py35_0  defaults
                             4.1.0                    py34_0  defaults
                             4.1.0                    py35_0  defaults
                             4.1.0                    py27_0  defaults
                             5.0.1                    py34_0  defaults
                             5.0.1                    py35_0  defaults
                             5.0.1                    py27_0  defaults
                             5.1.0                    py34_0  defaults
                             5.1.0                    py27_0  defaults
                             5.1.0                    py35_0  defaults
                             5.1.1                    py34_0  defaults
                             5.1.1                    py35_0  defaults
                             5.1.1                    py27_0  defaults
                          *  5.2.0                    py35_0  defaults
                             5.2.0                    py27_0  defaults
                             5.2.0                    py36_0  defaults
                             5.2.0                    py34_0  defaults
```
[Here](http://lpaste.net/402516410766458880) is my conda list output. Any idea on what is going on?

**Comment by minrk** [comment]
I guess Continuum hasn't updated the ipyparallel package on the defaults channel. It is updated on conda-forge, so you can get it with:

    conda install -c conda-forge ipyparallel

**Comment by ssoftcheck** [comment]
I have ipyparallel 6.0.2 and have a similar problem
[W 19:40:56.068 NotebookApp] 403 POST /clusters/default/start (::1): '_xsrf' argument missing from POST

**Comment by jonathanng** [comment]
I'm having the same issue.

**Comment by ssoftcheck** [comment]
I cleared out all configuration files after updating ipyparallel and that resolved the problem. However, when I stop clusters via the notebook interface it kills the entire jupyter-notebook process. I am not sure if this is intentional or desired behavior.

**Comment by minrk** [comment]
> However, when I stop clusters via the notebook interface it kills the entire jupyter-notebook process. I am not sure if this is intentional or desired behavior.

It sure isn't! I'll look into that one. Do you see anything in the log output?

**Comment by ssoftcheck** [comment]
>jupyter-notebook
[I 13:52:33.337 NotebookApp] The port 8888 is already in use, trying another port.
[I 13:52:33.359 NotebookApp] Loading IPython parallel extension
[I 13:52:33.484 NotebookApp] Serving notebooks from local directory: C:\Users\Sean
[I 13:52:33.484 NotebookApp] 0 active kernels
[I 13:52:33.484 NotebookApp] The Jupyter Notebook is running at: http://localhost:8889/?token=9d273e9bef5640740939b900c5810ccbee28e1e67b5da3f6
[I 13:52:33.484 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 13:52:33.485 NotebookApp]

    Copy/paste this URL into your browser when you connect for the first time,
    to login with a token:
        http://localhost:8889/?token=9d273e9bef5640740939b900c5810ccbee28e1e67b5da3f6
[I 13:52:33.639 NotebookApp] Accepting one-time-token-authenticated connection from ::1
[I 13:52:42.272 NotebookApp] Using existing profile dir: 'C:\\Users\\Sean\\.ipython\\profile_default'
[I 13:53:17.964 NotebookApp] Interrupted...
[I 13:53:17.966 NotebookApp] Shutting down kernels

The notebook process does successfully spawn the worker processes, but stopping them in the notebook interface shuts down the master process also

**Comment by mihkhub** [comment]
I have ipyparallel 6.0.2 and have a similar problem.
$ conda list ipyparallel
ipyparallel               6.0.2                    py27_0 

$ jupyter notebook
[I 09:40:03.460 NotebookApp] [nb_conda_kernels] enabled, 3 kernels found
[I 09:40:04.010 NotebookApp] Loading IPython parallel extension
[I 09:40:04.960 NotebookApp] [nb_anacondacloud] enabled
[I 09:40:04.964 NotebookApp] [nb_conda] enabled
[I 09:40:05.011 NotebookApp] ✓ nbpresent HTML export ENABLED
[W 09:40:05.011 NotebookApp] ✗ nbpresent PDF export DISABLED: No module named nbbrowserpdf.exporters.pdf
[I 09:40:05.017 NotebookApp] Serving notebooks from local directory: /home/atqs
[I 09:40:05.018 NotebookApp] 0 active kernels 
[I 09:40:05.018 NotebookApp] The Jupyter Notebook is running at: https://[all ip addresses on your system]:8888/
[I 09:40:05.018 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[W 09:40:34.280 NotebookApp] 403 POST /clusters/default/start (192.168.0.102): '_xsrf' argument missing from POST
[W 09:40:34.287 NotebookApp] 403 POST /clusters/default/start (192.168.0.102) 7.60ms referer=https://192.168.0.107:8888/tree?

**Comment by minrk** [comment]
this is fixed in the updated extension in 7.0
